cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java

```diff
@@ -60,19 +63,29 @@
  */
 public final class TokenSecureStoreUpdater implements SecureStoreUpdater {
   private static final Logger LOG = LoggerFactory.getLogger(TokenSecureStoreUpdater.class);
+
   private final YarnConfiguration hConf;
   private final LocationFactory locationFactory;
+  private final SecureStore secureStore;
   private final long updateInterval;
   private final boolean secureExplore;
+  private final boolean isKMSBacked;
 
   @Inject
-  public TokenSecureStoreUpdater(YarnConfiguration hConf, CConfiguration cConf, LocationFactory locationFactory) {
+  public TokenSecureStoreUpdater(YarnConfiguration hConf, CConfiguration cConf,
```

Comment:
[ERROR] /Users/Nishith/src/cdap/cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedFlowProgramRunner.java:[34,29] co.cask.cdap.security.TokenSecureStoreUpdater is not public in co.cask.cdap.security; cannot be accessed from outside package
[ERROR] /Users/Nishith/src/cdap/cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedFlowProgramRunner.java:[66,32] cannot find symbol
[ERROR] symbol:   class TokenSecureStoreUpdater